.github/workflows/verify-i18n.yml

```diff
@@ -0,0 +1,35 @@
+name: Verify i18n
+on:
+  workflow_call:
+
+permissions: {}
+
+jobs:
+  verify-i18n:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          persist-credentials: false
+
+      - uses: actions/setup-node@v4
+        with:
+          node-version-file: '.nvmrc'
+
+      - run: yarn install --immutable --check-cache
```

Comment:
fine for now, but this assumes to the repo uses `yarn`, and requires a full `yarn install` to run this step. 

Later we might want to re-jigger this somehow